src/cli/rustup_mode.rs

```diff
@@ -89,7 +90,7 @@ where
 pub fn main() -> Result<utils::ExitCode> {
     self_update::cleanup_self_updater()?;
 
-    use clap::ErrorKind::*;
+    use clap::error::ErrorKind::*;
```

Comment:
`clap::error::ErrorKind` is getting a bit verbose/deep. Perhaps 

```suggestion
    use clap::error::{ErrorKind::*, ErrorKind as ClapErrorKind};
```